tests/cases/elements/component_container_size.slint

```diff
@@ -0,0 +1,124 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+// FIXME: Skip embedding test on C++ and NodeJS since ComponentFactory is not
+// implemented there!
+//ignore: cpp,js
+
+import { Button } from "std-widgets.slint";
+
+export component TestCase inherits Rectangle {
+    in property <component-factory> cc <=> cc.component-factory;
+    out property <length> container-x <=> cc.x;
+    out property <length> container-y <=> cc.y;
+    in-out property <length> container-width <=> cc.width;
+    in-out property <length> container-height <=> cc.height;
+
+    width: 500px;
+    height: 500px;
+
+    cc := ComponentContainer {
+        x: 15px;
+        y: 20px;
+    }
+
+}
```

Comment:
This looks like it is the same as the previous one.  They can be merged into one  (you can have several '''rust  fences in one file)
Then it's faster because the slint part is only compiled once